pages/forums/view.js

```diff
@@ -0,0 +1,131 @@
+// Module imports
+import React from 'react'
+
+
+
+
+
+// Component imports
+import Component from '../../components/Component'
+import ForumThreadCard from '../../components/Forums/ForumThreadCard'
+import Page from '../../components/Page'
+
+
+
+
+
+// Component constants
+const pageTitle = 'Public Forums'
+
+
+
+
+
+class ForumList extends Component {
+  /***************************************************************************\
+    Private Methods
+  \***************************************************************************/
```

Comment:
But... There are no private methods? 😜

Reply:
oh you're just as bad with commenting the different import sections 😋